packages/libs/react-ui/src/styles/sprinkles.css.ts

```diff
@@ -102,6 +102,7 @@ const responsiveProperties = defineProperties({
     minWidth: [0],
     fontSize: vars.fontSizes,
     fontWeight: vars.fontWeights,
+    whiteSpace: ['nowrap'],
```

Comment:
can you add the other values for whitespace as well?
Than other people can use it as a sprinkle. or the common ones. 
```
whiteSpace: ['nowrap', 'break-spaces', 'normal']
```